packages/plugins/plugin-kanban/src/components/KanbanContainer.tsx

```diff
@@ -43,7 +43,7 @@ export const KanbanContainer = ({ object }: { object: Kanban.Kanban; role: strin
       );
       return unsubscribe;
     }
-  }, [typename, space]);
+  }, [schemas, typename, space]);
```

Comment:
i like to order these by "heft" (most to least important, or natural in the hierarchy, or least likely to change): schemas, space, typename